app/views/catalog/_index_gallery_collection_wrapper.html.erb

```diff
@@ -1,5 +1,5 @@
-<%# OVERRIDE Hyrax 3.4.1: make collection thumbnail a link so it matches works %>
-<div class="document col-xs-6 col-md-3">
+<%# OVERRIDE Hyrax v5.0.0rc2: make collection thumbnail a link so it matches works %>
+<div class="document col-6 col-md-3">
```

Comment:
Hmmm…from `col-xs-6` to `col-6`…curious.

Reply:
Boostrap 4 doesn't seem to have the `xs` size anymore.
https://getbootstrap.com/docs/4.0/layout/grid/#grid-options